### PR TITLE
Completed alert

### PR DESCRIPTION
**Pull request by nishasen**

### CODE REVIEW

File: Stylesheet/Components/alert.css [hunk 1]
```diff
@@ -1,2 +1,36 @@
 /* This css file is for alerts */
 
+.alert-box{
+    max-width: 90%;
+    padding: 0.5rem;
```
Comment (ttahm3d) [inline Stylesheet/Components/alert.css:6]: whenever you are doing `max-width /width: X% or Xvw` add `margin; 0 auto` to center it within container

File: playground.html [hunk 2]
```diff
@@ -45,5 +42,69 @@
...
+        This is a success alert!!!
+        <a class="btn-close">
+            <i class="fa fa-remove"></i>
+        </a>
+    </div>
     
- -->
\ No newline at end of file
+ -->
+
```
Comment (ttahm3d) [inline playground.html:85]: You might want to delete this from here? :)

File: Stylesheet/Components/alert.css (hunk 1 above)
Comment (nishasen) [reply Stylesheet/Components/alert.css:6]: Okay, doing that

File: playground.html (hunk 2 above)
Comment (nishasen) [reply playground.html:85]: Yes, looking into it

**Review by kjais1720 — COMMENTED** [review]
Good that you are using variables and not hard coded values for most properties👍

**Comment by sadathshariff** [comment]
Everything looks good, you can look into the suggestions by Tahir

**Comment by nishasen** [comment]
Yes doing that

**Pull request merged**